src/OpenCL/Silk.NET.OpenCL/Enums/MemProperties.gen.cs

```diff
@@ -7,13 +7,13 @@
 
 #pragma warning disable 1591
 
-namespace Silk.NET.OpenCL.Extensions.IMG
+namespace Silk.NET.OpenCL
```

Comment:
How come this has moved to the main package?

Reply:
well, it being not in the main package was a bug - it is defined as an empty enum in the base spec and extensions add stuff to it.

Reply:
ok